en/engine/event/event-node.md

```diff
@@ -0,0 +1,244 @@
+# Node Event System
+
+As mentioned in the previous document, the `input` object supports the global [input event system](event-input.md), and the `Node` also implements the event listening interface of the `EventTarget`. On this basis, we provide a basic node-related system events. This document will introduce how to use these events.
+
+__Cocos Creator__ supports four types of system events: __mouse__, __touch__, __keyboard__, __device motion__. They are called [Global Input Events](event-input.md). The usage of touch and mouse events dispatched by `Node` is discussed in this document. The usage is as follows:
+
+```ts
+node.on(Node.EventType.MOUSE_DOWN, (event) => {
+  console.log('Mouse down');
+}, this);
+```
+
+> __Note__: We no longer recommend directly using the event name string to register event listeners. Like the example above, please do not use `node.on('mouse-down', callback, target)` to register event listeners.
```

Comment:
```suggestion
> __Note__: It is no longer recommend directly using the event name string to register event listeners. Like the example above, please do not use `node.on('mouse-down', callback, target)` to register event listeners.
```

Reply:
I change in this way https://github.com/cocos-creator/creator-docs/pull/1903/commits/1f4a467d00b8b7e616c55458509fc0425af16d56